libcasr/src/stacktrace.rs

```diff
@@ -243,13 +243,49 @@ pub fn cluster_stacktraces(stacktraces: &[Stacktrace]) -> Result<Vec<u32>> {
     let mut flat_clusters = vec![0; len];
     for (i, (_, nums)) in clusters.into_iter().enumerate() {
         for num in nums {
-            flat_clusters[num] = i as u32 + 1; // Number clusters from 1, not 0
+            flat_clusters[num] = i + 1; // Number clusters from 1, not 0
         }
     }
 
     Ok(flat_clusters)
 }
 
+/// Perform crashline deduplication for each cluster
+///
+/// # Arguments
+///
+/// * `crashlines` - slice of crashlines as String
+///
+/// * 'clusters' - An vector of the same length as `crashlines`.
+/// Vec\[i\] is the flat cluster number to which original casrep i belongs.
+///
+/// * cluster_num - number of cluster
+///
+/// # Return value
+///
+/// An vector of the same length as `crashlines`.
```

Comment:
Maybe just say that size is equal to original `clusters` size?

Reply:
Done